src/Logging/Logging.Debug/Extensions/LogServiceFactory.Debug.Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CodeMonkeys.Logging.Debug;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceFactoryExtensions
+    {
+        /// <summary>
+        /// Method for adding the <see cref="ServiceProvider"/> to the <see cref="LogService"/>.
+        /// </summary>
+        public static void AddDebug(this ILogServiceFactory @this)
```

Comment:
Same

You seem to prefer not adding a line break for just one parameter.
We can discuss this.
Ignore these comments then.